### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by ozgur00**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -11,6 +11,17 @@
 ## Added
 - `showPreselectedStoredPaymentMethod` flag to `DropInConfiguration` to allow choosing to skip to the payment method selection screen.
 - Any `Configuration` will check if the `clientKey` matches the `environment` it will be used on.
+- Postal Code input field to Card Component.
+- KCP Authentication input field support to Card Component.
+- Social Security Number (CPF/CNPJ) input field support to Card Component.
+- Android 12 support.
+
+## Changed
+- Visibility of Expiry Date in Card Component changed to an enum with REQUIRED, OPTIONAL and HIDDEN as options.
```
Comment (caiofaustino) [inline RELEASE_NOTES.md:20]: This is an internal change and doesn't have any effect for the merchant so no need to add here.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -11,6 +11,17 @@
...
+- Adyen 3DS2 SDK version to 2.2.4.
+
+## Removed
+- Switching to next input field automatically after card number has been filled in.
 
 ## Fixed
 - Logos not loading when multiple icons in the same screen have the same logo.
```
Comment (caiofaustino) [inline RELEASE_NOTES.md:27]: - Card number validation changed for cards where the number doesn't pass the Luhn check but is still valid.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (ozgur00) [reply RELEASE_NOTES.md]: removed.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (ozgur00) [reply RELEASE_NOTES.md:26]: added 👍

**Review by caiofaustino — APPROVED** [review]

**Pull request merged**